### ISSUE TITLE
Not all tags supported (C#)

### ISSUE THREAD

**Issue by CherylWriter** [oleg-shilo/DocPreview.VSIX#13]
This is a useful tool. Thank you! I do notice that it doesn't properly support all of the tags, such as seealso, value, and note. It would be an even more useful tool if you could support the tags listed in the Sandcastle Wall Chart (you can search for it online). Thanks again!

**Comment by CherylWriter** [comment]
Or at minimum the tags listed in the Microsoft guide:
Recommended Tags for Documentation Comments (C# Programming Guide)
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/recommended-tags-for-documentation-comments

Thanks!

**Comment by oleg-shilo** [comment]
Will see what can be done.
Txs

**Comment by oleg-shilo** [comment]
When the extension was developed the the target was to achieve the  of the recommended tags for Documentation Comments (C# Programming Guide). Exactly the same link as in teh second message in this thread ([tags](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/recommended-tags-for-documentation-comments)).

All except `<include>`. This one is outside of the scope of this project.

I have just rechecked and all the tags are supported. Did you find something that I missed?

**Comment by win32nipuh** [comment]
I have the xml comment like this:

```
 /// <summary>
        /// Create the new object from existing object.
        /// </summary>
        /// <remarks>
        /// Now it is possible to create.
        /// <list type="bullet">
        /// <term>Master object from a Slave object</term>
        /// <term>Master object from other Master object</term>
        /// <term>Document from Document</term>
        /// </list>
        /// </remarks>
        /// <param name="request">
        /// The <see cref="MessageContracts.CreateObjectByObjectMessageRequest"/> object.
        /// </param>
        /// <returns>
        /// The <see cref="MessageContracts.CreateObjectMessageResponse"/> object. 
        /// </returns>

```
But all what inside of /// <list type="bullet">  is ignored

**Comment by oleg-shilo** [comment]
Seems to work. 

Ensure you have refreshed the preview with the refresh button:
![image](https://user-images.githubusercontent.com/16729806/75350011-e3319580-58f9-11ea-8bf7-3c0ffb41796a.png)

And of course for the bullet items you need to use these tags:
```c#
    /// <item><description>Master object from a Slave object</description></item>
    /// <item><description>Master object from other Master object</description></item>
    /// <item><description>Document from Document</description></item>
```

**Comment by akalnay** [comment]
Is the `<inheritdoc>` tag supported?  It is listed on the [Tags](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/recommended-tags-for-documentation-comments#tags) page.  One of the comments above indicates that except for `<include>` all the tags in that page are covered, however I can't get `<inheritdoc>` to work with `DocPreview`.

**Comment by oleg-shilo** [comment]
`<inheritdoc>` is a tag that was introduced only this year (at least documented):
![image](https://user-images.githubusercontent.com/16729806/93213958-e2822300-f7a7-11ea-857a-fae24fa4274b.png)

So it's not supported. Though in this case the need for support can be questioned. As it does require a significant extension to the framework (building the type tree for the whole project) and yet the benefit is highly debatable. Thus displaying the help content without being able to edit it not really that useful.

**Comment by akalnay** [comment]
What I was looking forward to doing is something similar to what is shown in the [Sandcastle documentation examples](https://ewsoftware.github.io/XMLCommentsGuide/html/86453FFB-B978-4A2A-9EB5-70E118CA8073.htm#Examples).  Specifically, something like this example towards the bottom of that page:

```C#
     #pragma warning disable 1573
     /// <inheritdoc cref="OverloadedMethod(string)" />
     /// <param name="p2">The second string parameter</param>
     public void OverloadedMethod(string p1, string p2)
     {
         // Inherit documentation from the first overload and add
         // comments for the second parameter.
 
         // Note that because we supplied comments for one parameter
         // but not the other, the compiler will complain.  However,
         // we can shut it up by using a "#pragma warning" directive as
         // shown.
     }
```
For that example, I would not care about not being able to edit the inherited documentation.  That is actually the point as I don't want to have to change the inherited documentation, I only want to be able to add documentation for the added parameter and leave everything else as is without having to retype (or copy) the original documentation and not being concerned with keeping the documentation for the overloaded methods in sync.

**Comment by oleg-shilo** [comment]
OK, I see your point.

Unfortunately it is difficult to deliver this feature without rearchitecturing the extension. 

Currently it simply takes the XML comment section and converts it into the HTML snippet. In order to implement your enhancement the extension will need to keep track of all XML sections in your project as well as the ones that come from referenced assemblies.

It is doable but a big task. I am marking the issue as "help wanted" and  "voting" so people can up-vote it and possibly start contributing.

**Comment by akalnay** [comment]
Thank you @oleg-shilo.  I hope you don't mind that I created a separate issue ([Add support for the <inheritdoc\> tag #17](https://github.com/oleg-shilo/DocPreview.VSIX/issues/17)) to track the possibility of adding support for `<inheritdoc>`.  This was with the intent of giving this possible change more visibility and hopefully more chances of being voted on and supported.  Otherwise it risks being buried and lost in the comments for the [Not all tags supported (C#) #13](https://github.com/oleg-shilo/DocPreview.VSIX/issues/13) issue.  I also created the issue to make it clear that the possible enhancement would be specific to `<inheritdoc>` and not include the `<include>` tag which you mentioned as being out of scope in a comment above.  Perhaps the `help wanted` and `voting` tags can be moved to the new issue?

Thank you!

**Comment by oleg-shilo** [comment]
Agree. It does make sense to move it into an individual feature request.

**Comment by oleg-shilo** [comment]
Done in the v1.6.0 release
